### ISSUE TITLE
Entity to Grid - Export Control User Limit

### ISSUE THREAD

**Issue by matomatusov** [slipx06/sunsynk-power-flow-card#194]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

addition to the Grid

### Expected behaviour

Hi,
would it be possible to add an entity to the Grid section?
It would be about Export Control User Limit (W). 

![exp  control](https://github.com/slipx06/sunsynk-power-flow-card/assets/129114957/328ae845-0d74-4f3f-bf9b-43237ca00747)

Thanks :)

### Mode

both

### Context / Reason

It would be good information about setting up export to the Grid.

**Comment by slipx06** [comment]
The equivalent Synsynk setting is "max sell power". Will need to see if there is space for this on the card

**Comment by jzielke84** [comment]
@slipx06 Just to be sure before I open a new issue: Is this also required if I'm injecting power back into the grid? This is my card configuration:

```
yaml
type: custom:sunsynk-power-flow-card
cardstyle: compact
show_solar: true
battery:
  energy: 0
  shutdown_soc: 20
  show_daily: false
solar:
  show_daily: false
  mppts: 1
  max_power: 800
  dynamic_colour: true
  pv1_name: Solix
  auto_scale: true
load:
  show_daily: false
  auto_scale: true
  dynamic_colour: true
  dynamic_icon: true
grid:
  show_daily_buy: false
  show_daily_sell: false
  energy_cost_decimals: 0
  auto_scale: true
entities:
  pv1_power_186: sensor.solix_power
  essential_power: sensor.energie_eigenverbruch
  grid_voltage: sensor.bitshake_smartmeterreader_sml_volt_p2
  grid_ct_power_172: sensor.netz_nettobezug
  solar_sell_247: sensor.solix_netzeinspeisung
large_font: true
panel_mode: true
show_grid: true
show_battery: false
decimal_places: 2
inverter:
  three_phase: false
  auto_scale: true
title: Stromverbrauch
title_size: '1'
```

The calculations are fine but the grid should also show a value for injecting back into the grid. Like a dot flowing from right to left. This is my current output:

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/13448042/b2cab6a6-7d99-4d45-8fc7-77162eb3a85a)

**Comment by slipx06** [comment]
Hi. No its not required. The card will take care of that if your grid sensor i.e. `grid_ct_power_172: sensor.netz_nettobezug` reports positive power for import and negative power for export.

**Comment by jzielke84** [comment]
> Hi. No its not required. The card will take care of that if your grid sensor i.e. `grid_ct_power_172: sensor.netz_nettobezug` reports positive power for import and negative power for export.

Ah ok, thank you for clarifying this one.

**Comment by slipx06** [comment]
@matomatusov finally got around to adding this feature request to https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.29.0

**Comment by matomatusov** [comment]
HI, 
I updated and I can't find max_sell_power in the entites.
Is it in the endites?

**Comment by slipx06** [comment]
Sorry I have not added it to the visual editor yet. Will push a fix. For now you can add it under the entities: using the code editor
``
entities:
   max_sell_power: number.sunsynk_max_sell_power
```

**Comment by matomatusov** [comment]
I have it there thanks
![2](https://github.com/slipx06/sunsynk-power-flow-card/assets/129114957/ab6afc0d-6e23-4e25-b5ed-75520f5ffdf7)

**Comment by slipx06** [comment]
Its added to https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.29.1

**Comment by slipx06** [comment]
Fixed in 4.29.1

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/337e30d1-9077-49de-ab5f-031f30d96610)

**Comment by matomatusov** [comment]
I confirm it's in the visual editor (max_sell_power). 
The export is also correct.
